pkg_resources/__init__.py

```diff
@@ -3295,8 +3287,8 @@ def check_version_conflict(self):
             ):
                 continue
             issue_warning(
-                "Module %s was already imported from %s, but %s is being added"
-                " to sys.path" % (modname, fn, self.location),
+                f"Module {modname} was already imported from {fn}, but {self.location} is being added"
+                " to sys.path",
```

Comment:
I'd either split the line back under the line limit. Or merge it back since rule E501 isn't enabled.

Currently it's the worst of both worlds (102 characters and split)

Reply:
I chose to merge it back, in the spirit of the [Linux kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html#breaking-long-lines-and-strings):
> However, never break user-visible strings such as printk messages, because that breaks the ability to grep for them.